code/modules/clothing/neck/_neck.dm

```diff
@@ -180,13 +180,19 @@
 	icon_state = "stripedbluescarf"
 	item_color = "stripedbluescarf"
 
-/obj/item/clothing/neck/petcollar //don't really wear this though please c'mon seriously guys
+/obj/item/clothing/neck/petcollar
 	name = "pet collar"
-	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule."
+	desc = "It's for pets."
 	icon_state = "petcollar"
 	item_color = "petcollar"
 	var/tagname = null
 
+/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
+	if(ishuman(M))
```

Comment:
```suggestion
	if(ishumanbasic(M))
```

https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/is_helpers.dm#L69

Reply:
no

Reply:
It's the same thing as is currently but with the proper define.

I dunno why lizards and the like can still wear it but lol whatever.